src/metabase/api/search.clj

```diff
@@ -477,26 +448,39 @@
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
 (mu/defn ^:private search-context :- SearchContext
-  [search-string   :- [:maybe ms/NonBlankString]
-   archived-string :- [:maybe ms/BooleanString]
-   table-db-id     :- [:maybe ms/PositiveInt]
-   models          :- [:maybe [:or SearchableModel [:sequential SearchableModel]]]
-   limit           :- [:maybe ms/PositiveInt]
-   offset          :- [:maybe ms/IntGreaterThanOrEqualToZero]]
+  [{:keys [search-string
+           archived-string
+           created-by
+           table-db-id
+           models
+           limit
+           offset]} :- [:map {:closed true}
```

Comment:
Can you order these alphabetically while we're at it, I'm assuming we'll add a lot of things to this list

Reply:
done